packages/web-pkg/src/helpers/preview/textFilePreview.ts

```diff
@@ -0,0 +1,6 @@
+import { Resource } from 'web-client/src'
+
+export const isResourceTxtFileAlmostEmpty = (resource: Resource): boolean => {
```

Comment:
my last two nitpicks:
- this is a resource helper that has nothing to do with previews anymore. The file name and location should reflect that accordingly. This only states if a text file is almost empty. Nothing more, nothing less.
- please add a unit test for the helper

Reply:
yup